java/runtime/src/main/resources/ray.default.conf

```diff
@@ -25,15 +25,21 @@ ray {
   // Available resources on this node, for example "CPU:4,GPU:0".
   resources: ""
 
-    // Configuration items about driver.
-    driver {
-      // If worker.mode is DRIVER, specify the driver id.
-      // If not provided, a random id will be used.
-      id: ""
-      // If worker.mode is WORKER, it means that worker will load
-      // the resources from this path to execute tasks.
-      resource-path: /tmp/ray/driver/resource
+  // Configuration items about driver.
+  driver {
+    // If worker.mode is DRIVER, specify the driver id.
+    // If not provided, a random id will be used.
+    id: ""
+
+    // If worker.mode is WORKER, this field will be enabled.
+    // And if worker.mode is  DRIVER, this field will not.
+    resource {
+      // Whether the worker should load resources from local path.
+      from-local: false
```

Comment:
This PR looks good to me overall. But I think the naming is confusing. What this flag really means is whether each driver should load resources from its own path (instead of all driver loading resources from the same path). Thus, `from-local` doesn't express the exact meaning.

After thinking more about it, I think it might be better to just use one config `ray.driver.resource.path`. And we can document this config like this: `If this config is set, worker will use different paths to load resources when executing tasks from different drivers. E.g., if it's set to '/tmp/driver_resource', the path for driver 123 will be '/tmp/driver_resource/123'`